lib/samson/performance_tracer.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+module Samson
+  module PerformanceTracer
+    # It's used to trace the hook fire event,
+    # We can't use the samson hook to fetch the available plugins.
+    TRACER_PLUGINS = ['SamsonNewRelic', 'SamsonDatadogTracer::APM'].freeze
```

Comment:
having the main code know about what plugins are available is kinda backwards ... should call a hook to get the list or do something like `plugins.map(&:tracer)`

Reply:
also avoids doing the constantize

Reply:
I have commented that we can' use hook, because this is used to trace the hook, this will create a endless loop.